packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -136,10 +140,22 @@ export class AggregatedAttestationPool {
       attestationGroupByIndex = new Map<CommitteeIndex, MatchingDataAttestationGroup>();
       attestationGroupByIndexByDataHash.set(dataRootHex, attestationGroupByIndex);
     }
-    const committeeIndex = isElectraAttestation(attestation)
-      ? // this attestation is added to pool after validation
-        attestation.committeeBits.getSingleTrueBit()
-      : attestation.data.index;
+
+    let committeeIndex;
+
+    if (fork >= ForkSeq.electra) {
```

Comment:
to make it more obvious that fork is based on slot passed in and not attestation slot

```suggestion
    if (this.config.getForkSeq(slot) >= ForkSeq.electra) {
```